.github/pull_request_template.md

```diff
@@ -0,0 +1,34 @@
+Checklist:
+
+- Does this pull request close an issue? We encourage you to open an issue first if this pull request (PR) is not a
+  minor change.
+  - [ ] no
+  - [ ] yes
+    - Close #{issue_number}
+
+For the following questions, only check the boxes that are applicable.
+
+- [ ] Add descriptive pull request text.
+- [ ] Add/update relevant tests.
+- [ ] Add/update relevant documents.
+
+- Do you have triage permission of this repository?
+  - [ ] no
+  - [ ] yes
+    - Does this pull request close an issue?
+      - [ ] No, this pull request also acts as an issue by itself.
+        - [ ] Assign yourself.
+        - [ ] Label this pull request.
+        - [ ] Put a milestone.
+        - [ ] Put this pull request under "In Progress" or "Under Review" pipeline on ZenHub.
+        - [ ] Put this pull request under the appropriate epic on ZenHub.
+        - [ ] Put an estimate on ZenHub.
+        - [ ] Add dependency relationship with other issues/pull requests on ZenHub.
+      - [ ] Yes, this pull request addresses an issue and does not act as an issue.
+        - [ ] Connect this pull request to the underlying issue on ZenHub.
+        - [ ] DON'T assign yourself or anyone else.
+        - [ ] DON'T label.
+        - [ ] DON'T put a milestone.
+        - [ ] DON'T put this pull request under any epic on ZenHub.
+        - [ ] DON'T put an estimate on ZenHub.
```

Comment:
Cosmetically should we combine this into one line. The PR comment is going to be large, but not a huge concern.

Reply:
How about the new version?

Reply:
Don't think it addresses the cosmetic concern since everything is still on separate lines, but it's not a big deal. 

Who would have triage permission for this repo, just CODAIT team members?

Reply:
No, currently only you, me and Julia have. We should probably give everyone the triage permission for scrum purposes.

Reply:
I updated the permission. I don't know how to format these lines to be cosmetically reasonable though. I added a horizontal line at the top, which hopefully would be a good separator between the description of the PR and this checklist.

Reply:
How do you feel about enclosing the checklist in a dropdown so that it can be hidden when we don't need to look at the answers?

Reply:
By everyone you mean the CODAIT team? We probably wouldn't use Zenhub to track external PR's right, or would we?

Reply:
Why not, it's a way to see how much contribution is from external people. We can do the ZenHub labor for them.

Reply:
Gotcha, ok sounds good then.

Reply:
Did you see the message above about the dropdown?

> How do you feel about enclosing the checklist in a dropdown so that it can be hidden when we don't need to look at the answers?

Reply:
Sorry I missed that message. I added a dropdown. It seems like aesthetically this is the best I can do (you can't have the section header to the right of the triangle).

Reply:
Thanks, looks good